Gini-iOS-SDK/GINIAPIManager.h

```diff
@@ -90,6 +133,17 @@ typedef NS_ENUM(NSUInteger, GiniAPIResponseType){
  */
 - (BFTask *)getPagesForDocument:(NSString *)documentId;
 
+/**
+ * Gets the list of pages for a document.
```

Comment:
It is not only this method but the whole library, since it always returns a task that returns something. Sometimes even another task.
Do you think we should tackler this for this PR? I would like to improve the documentation in general so I guess we can do it in that moment.

Reply:
Ok, make an issue and work on it separately. Or change the new method docs. Your choice.